pkgs/development/compilers/gcc/common/configure-flags.nix

```diff
@@ -139,8 +139,8 @@ let
       }"
     ]
 
-    ++ (if (enableMultilib || targetPlatform.isAvr)
-      then ["--enable-multilib" "--disable-libquadmath"]
+    ++ (if (enableMultilib || targetPlatform.isAvr || (targetPlatform.gcc.enableMultilib or false))
```

Comment:
Can we instead default `enableMultilib` based on this? (I know that would have to be per ghc in some caller leading to this file, but I think it's still better to be consistent.)

Reply:
yeah, that sounds like an improvement